app/src/components/Shared/FooterMini.jsx

```diff
@@ -34,11 +39,21 @@ class FooterMini extends Component {
         </div>
       );
     }
+    if (this.props.supportModalVisible) {
```

Comment:
why do we need two components here? we already have a modal implemented for login and share, cant we use somtehing like that ?